src/main.js

```diff
@@ -335,6 +335,33 @@ function signLogicSigTransaction(txn, lsig) {
     };
 }
 
+/**
+ * signLogicSigTransactionObject takes transaction.Transaction and a LogicSig object and returns a logicsig
+ * transaction which is a blob representing a transaction and logicsig object.
+ * @param {Object} txn transaction.Transaction
+ * @param {LogicSig} lsig logicsig object
+ * @returns {Object} Object containing txID and blob representing signed transaction.
+ */
+function signLogicSigTransactionObject(txn, lsig) {
```

Comment:
test for it?

Reply:
Deduplicated code so it's now included in the test